heroku.yml

```diff
@@ -0,0 +1,5 @@
+build:
+  docker:
+    web: Dockerfile
+run:
+  web: bash start.sh
```

Comment:
Since this needs start.sh, shouldn't it be in the same PR as https://github.com/Rudloff/alltube/pull/317?

Reply:
> Since this needs start.sh, shouldn't it be in the same PR as #317?

yes.thats right